backend/seeds/user_seeder.py

```
@@ -39,19 +50,30 @@ def seed_admin():
+            print("User exists, skipping seeding")
+            return
+        user = User(
+            username="user",
+            email="user@example.com",
+            name="Test User",
+            hashed_password=User.get_password_hash("test1234"),
+        )
+        db.add(user)
+        db.commit()
+        print("Test user created successfully!")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)